stdlib/runtime/numberUtils.gr

```diff
@@ -886,6 +886,25 @@ let utoa64_any_core = (buffer, num, offset, radix) => {
   }
 }
 
+export let utoa32Buf = (buf, value, radix) => {
```

Comment:
Could we instead call this `utoa32Buffered`? Reminiscing https://github.com/grain-lang/grain/issues/555.